packages/discovery-provider/src/queries/get_challenges.py

```diff
@@ -48,15 +48,24 @@ def rollup_aggregates(
     parent_challenge: Challenge,
     disbursements: List[ChallengeDisbursement],
 ) -> ChallengeResponse:
+    step_count = parent_challenge.step_count
     num_complete = reduce(
         lambda acc, cur: cast(int, acc) + cur.amount if cur.is_complete else acc,
         user_challenges,
         0,
     )
+    amount = parent_challenge.amount
 
     # The parent challenge should have a step count, otherwise, we can just
     # say it's complete.
-    if parent_challenge.step_count:
+    if parent_challenge.id == "o":
```

Comment:
OOTL - how/why is this different from the Audio match?

Reply:
audio match uses a different modal https://github.com/AudiusProject/audius-protocol/blob/main/packages/web/src/pages/audio-rewards-page/components/modals/ChallengeRewardsModal/ChallengeRewardsModal.tsx#L571-L579

and it's repeatable. there isn't really a challenge type that fits nicely w this one where it's unique to a user and only disbursed once. there needs to be some logic in the backend or client that handles this one differently. e